diff --git a/node_modules/expo-document-picker/android/build.gradle b/node_modules/expo-document-picker/android/build.gradle
index f57f6b6..b4819ec 100644
--- a/node_modules/expo-document-picker/android/build.gradle
+++ b/node_modules/expo-document-picker/android/build.gradle
@@ -27,20 +27,7 @@ task androidSourcesJar(type: Jar) {
 }
 
 afterEvaluate {
-  publishing {
-    publications {
-      release(MavenPublication) {
-        from components.release
-        // Add additional sourcesJar to artifacts
-        artifact(androidSourcesJar)
-      }
-    }
-    repositories {
-      maven {
-        url = mavenLocal().url
-      }
-    }
-  }
+  println("Dummy")
 }
 
 android {
